packages/chart/src/ChartUtils.ts

```diff
@@ -1877,12 +1855,67 @@ class ChartUtils {
     return axis;
   }
 
+  /**
+   * Creates a plotly layout template object based on a given theme.
+   * See https://plotly.com/javascript/reference/layout/#layout-template
+   * @param theme The theme to use for the layout template
+   */
+  makeDefaultTemplate(theme: Partial<ChartTheme>): Template {
```

Comment:
Add an `@returns`

Also, why is the param a partial? Shouldn't it just expect the full theme?

Reply:
I fixed the partial. Do we need a `@returns` here? I don't see that we consistently do this, and I don't see it adding any clarity to this particular util.